packages/qtx11extras.rb

```diff
@@ -28,8 +20,6 @@ def self.build
   end
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}"
-    system "cp -a lib/* #{CREW_DEST_LIB_PREFIX}"
```

Comment:
Why no libraries?

Reply:
It is not part of the tar file and not produced by the compilation step